deploy/docs/Non_Helm_Installation.md

```diff
@@ -202,12 +205,26 @@ __NOTE__ It’s fine to change the value of the `cluster` field, but don’t cha
 
 __NOTE__ If you plan to install Prometheus in a different namespace than you deployed Fluentd to in Step 1, or you have an existing Prometheus you plan to apply our configuration to running in a different namespace,  please update the remote write API configuration to use the full service URL like, `http://collection-sumologic.sumologic.svc.cluster.local:9888`.
 
-You can also [Filter metrics](#filter-metrics) and [Trim and relabel metrics](#trim-and-relabel-metrics) in `prometheus-overrides.yaml`.
+You can also [Filter metrics](additional_prometheus_configuration.md#filter-metrics) and [Trim and relabel metrics](additional_prometheus_configuration.md#trim-and-relabel-metrics) in `prometheus-overrides.yaml`.
 
 Install `prometheus-operator` by generating the yaml files using Helm:
 
 ```bash
-$ helm template stable/prometheus-operator --name prometheus-operator --set dryRun=true -f prometheus-overrides.yaml > prometheus.yaml
+<<<<<<< HEAD
```

Comment:
There's some merge conflicts here, I see we have the other `helm fetch` command on line 189 above that conflicts with line 214 here

Reply:
sigh, something is really messed up with my Git. Thanks @samjsong

Reply:
fixed